Source/Particles/Collision/BinaryCollision/BinaryCollision.H

```diff
@@ -199,6 +217,12 @@ public:
                     amrex::HostDevice::Atomic::Add( &(*cost)[mfi.index()], wt);
                 }
             }
+
+            if (m_have_product_species) {
+                // Call redistribute to remove particles with negative ids
+                species1.Redistribute(lev, lev, 0, true, true);
+                species2.Redistribute(lev, lev, 0, true, true);
```

Comment:
The `local` parameter sets how many cells particles could have moved since the previous `Redistribute` call. Therefore, a "local" redistribute has `local=0`.
```suggestion
                species1.Redistribute(lev, lev, 0, 0, true);
                species2.Redistribute(lev, lev, 0, 0, true);
```